src/definition/messages/IMessagePinContext.ts

```diff
@@ -0,0 +1,21 @@
+import { IMessage } from '.';
+import { IUser } from '../users';
+
+/**
+ * The context of execution for the following events:
+ * - IPostMessagePinned
+ */
+export interface IMessagePinContext {
+    /**
+     * The message that was pinned or unpinned
+     */
+    message: IMessage;
+    /**
+     * The user who pinned the message
+     */
+    user: IUser;
+    /**
+     * If the message was pinned or unpinned
+     */
+    isUnpinned: boolean;
```

Comment:
Same concept here, use the positive so when it is true then it is true instead of an inverse of negative.